src/charm.py

```diff
@@ -409,7 +409,8 @@ def _request_certificates(self):
         self.certificates.request_client_cert("system:kubelet")
 
     def update_status(self, _event):
-        """Handle the update status hook event.
+        """
+        Handle the update status hook event.
```

Comment:
Hmm. Something seems off with this new formatting, I thought the summary was always on the first line. Perhaps a new rule was introduced?

Reply:
maybe [these changes](https://github.com/charmed-kubernetes/charm-kubernetes-worker/pull/175/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L23-R21) caused `ruff` to adjust these